src/commands/MigrationRevertCommand.ts

```diff
@@ -45,7 +45,9 @@ export class MigrationRevertCommand {
         } catch (err) {
             console.error(err);
             // throw err;
+        } finally {
+            process.exit(0);
```

Comment:
I agree, but this helped me as a hotfix. If I'll have some more time I'll try to narrow the problem.